circle.yml

```diff
@@ -22,6 +22,23 @@ test:
     - "cd $GOPATH/src/github.com/SpectoLabs/hoverfly && make test"
 
 deployment:
+  master:
+    branch: master
+    commands:
+      - "cd $GOPATH/src/github.com/SpectoLabs/hoverfly && GIT_TAG_NAME=master-$CIRCLE_BUILD_NUM make build" 
```

Comment:
I'd like to try and keep all the build related stuff in the circle.yml. When we eventually upgrade to circle2, it has support for workflows this would move to a workflow task.